examples/send-multiple-l2-tx/main.go

```diff
@@ -0,0 +1,140 @@
+package main
+
+import (
+	"encoding/json"
+	"log"
+	"math/big"
+
+	"github.com/hermeznetwork/hermez-go-sdk/account"
+	"github.com/hermeznetwork/hermez-go-sdk/client"
+	"github.com/hermeznetwork/hermez-go-sdk/node"
+	"github.com/hermeznetwork/hermez-go-sdk/transaction"
+	hezCommon "github.com/hermeznetwork/hermez-node/common"
+)
+
+const (
+	ethereumNodeURL           = ""
+	chainID                   = 5 // set this accordingly to the ethereum node
+	auctionContractAddressHex = "0x1D5c3Dd2003118743D596D7DB7EA07de6C90fB20"
+
+	sourceAccPvtKey = ""
+
+	txsReceiverMetadataJson = `
+	[
+		{ "to_eth_addr": "0xb48cA794d49EeC406A5dD2c547717e37b5952a83", "fee_selector": 126, "amount": "1100000000000000000" },
+		{ "to_eth_addr": "0x263C3Ab7E4832eDF623fBdD66ACee71c028Ff591", "fee_selector": 126, "amount": "1200000000000000000" },
+		{ "to_eth_addr": "0xb8eD2B0a6e17649c9cE891895D3D9297Ab448f03", "fee_selector": 126, "amount": "1300000000000000000" },
+		{ "to_eth_addr": "0x4E857Ac4A07cAD0B50CD006158f5E5A521A880CE", "fee_selector": 126, "amount": "1400000000000000000" }
+	]`
+)
+
+var (
+	hezToken = hezCommon.Token{
+		TokenID: 1,
+		Symbol:  "HEZ",
+	}
+)
+
+func main() {
+	var debug bool = false
+
+	log.Println("Starting Hermez Client...")
+	hezClient, err := client.NewHermezClient(ethereumNodeURL, auctionContractAddressHex, chainID)
+	if err != nil {
+		log.Printf("Error during Hermez client initialization: %s\n", err.Error())
+		return
+	}
+	log.Println("Connected to Hermez Smart Contracts...")
+	log.Println("Pulling account info from a coordinator...")
+
+	bootCoordNodeState, err := node.GetBootCoordinatorNodeInfo(hezClient)
```

Comment:
Why are we using the boot coordinator?? Maybe it better to set the coordinartor url and port to make this tool more flexible

Reply:
Because it needs to find the actual coordinator. As we don't have shared transaction pool this mechanism saves time to the Dev trying to figure out to what coordinator she needs to send the transaction

Reply:
I get this point but if we want to try a specific feature, maybe it's better to allow specify the coordinator that we want